packages/client/src/runtime/core/init/getEngineInstance.ts

```diff
@@ -102,7 +106,5 @@ export function getEngineInstance({ copyEngine = true }: GetPrismaClientConfig,
     })
   }
 
-  throw new PrismaClientValidationError('Invalid client engine type, please use `library` or `binary`', {
-    clientVersion: engineConfig.clientVersion,
-  })
```

Comment:
This error will never happen again, it was confusing users. Instead, as a last resort, the engine is used accordingly to the client build/import, and if it must error then it is handled by the engine itself. For example, before this PR, if you used `@prisma/client/edge` with a regular url instead of an accelerate url, you'd get this cryptic error.